dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -220,6 +222,11 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
             if (!AppModel::Identity::IsPackagedProcess())
             {
+                // If the app icon was inferred from process, then we should clean it up.
+                // Do not fail this function if such a file doesn't exist.
```

Comment:
I think we should mention that the file may not exist in cache if the icon was retrieved from Shortcut or was the default icon. So this is more of best-effort check